pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1285,6 +1286,13 @@ private SchemaStorage createAndStartSchemaStorage() throws Exception {
         return schemaStorage;
     }
 
+    private void setSchemaRegistryName(SchemaStorage schemaStorage) {
```

Comment:
the name is not very clear
what about 'ensureSchemaRegistryClassName' ?

Reply:
sure, i have updated it